Update sage-bank-feeds-authenticate-users-web-app.md (#543)
codatio/codat-docs: docs/bank-feeds-api/sage-bank-feeds/sage-bank-feeds-authenticate-users-web-app.md

8. Your web app authenticates the user against the Codat company and data connection to which they are linked.

:::caution Iframe

> Sage will surface your pages within an iframe embedded within the Sage product, therefore you must ensure that your UI can be used within this format.

Reviewer: ```suggestion
Sage will display your pages within an iframe embedded in the Sage product. Ensure that your UI can be used within this format.
```

:::

> You must include the "Content-Security-Policy" header with a value of `frame-ancestors 'self' https://*.sagebankdrive.com https://*.sage.com https://*.intacct.com https://*.sageone.com` to ensure it functions within the iframe.

Reviewer: Does this sentence also need to be part of the admonition?
